### PR TITLE
docs: is active method in transaction

### PR DESCRIPTION
**Pull request by jannyHou**
Implements https://github.com/strongloop/loopback-next/issues/3471
The doc PR.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Transactions.md [hunk 1]
```diff
@@ -172,6 +172,24 @@ on transaction objects are as follows:
...
+  /**
+   * Check if the transaction has an active connection
+   */
+  isActive(): boolean;
+```
+
+Suppose you have a transaction called `tx`, you can call `tx.isActive()` to
+check whether it's still active.
+
+{% include note.html content=" The postgresql connector doesn't fully support this function due to its special way of creating the transaction instance. Details are explained in [issue #411](https://github.com/strongloop/loopback-connector-postgresql/issues/411). " %}
```
Comment (dhmlau) [inline docs/site/Transactions.md]: Could you please elaborate on "The postgresql connector doesn't fully support this function"? 
And there's an extra space after the quote.

Comment (jannyHou) [reply docs/site/Transactions.md]: @dhmlau The details are explained in [issue #411](https://github.com/strongloop/loopback-connector-postgresql/issues/411), or do you want me to expand the content here?

The function itself is simple, and I already have a brief explanation for the inconsistency:
> The postgresql connector doesn't fully support this function due to its special way of creating the transaction instance.

Comment (dhmlau) [reply docs/site/Transactions.md]: I'm thinking whether we could add a one-liner about what doesn't really work in the postgresql connector, and then point to the issue for more details.

Comment (jannyHou) [reply docs/site/Transactions.md]: @dhmlau added

**Pull request merged**